tests/test-config.js

```diff
@@ -20,7 +23,7 @@ const getLocal = () => {
   if(process.env.LOCAL_ONLY != undefined) {
     return convertToBoolean(process.env.LOCAL_ONLY);
   }
-  return _runner.local;
+  return !localSettings.enableInteropTests;
```

Comment:
this doesn't match the name in the example in the README! I do like this though and good job on this PR.

Reply:
Yeah...I forgot to revisit the README. Will do that now.